app/src/main/java/com/ndt/themoviedb/ui/base/BaseViewHolder.kt

```diff
@@ -0,0 +1,22 @@
+package com.ndt.themoviedb.ui.base
+
+import android.view.View
+import androidx.recyclerview.widget.RecyclerView
+
+open class BaseViewHolder<T>(itemView: View, private var onItemClick: (T, Int) -> Unit) :
```

Comment:
```suggestion
open class BaseViewHolder<T>(itemView: View, private val onItemClick: (T, Int) -> Unit) :
```
thế này chứ đúng ko nhỉ ?